### ISSUE TITLE
attached files (data "dir")

### ISSUE THREAD

**Issue by marcinofulus** [vbraun/ExportSageNB#14]
Files from "data dir attached to sagenb notebooks are not saved.

There is a solution in sws2rst, namely for each name there is a directory created with _files suffix and 
internal links are also updated.

**Comment by kcrisman** [comment]
Correct, this was one of the original issues I raised about .ipynb because there seems to not really be an equivalent notion.

**Comment by jdemeyer** [comment]
The "equivalent" notion for Jupyter would be just ordinary files on the file system.

**Comment by kcrisman** [comment]
Sure, but what I mean is how would one bundle this up as an export?
